papaya-web/src/main/java/pl/edu/agh/papaya/rest/projects/controller/ProjectsController.java

```diff
@@ -104,6 +110,39 @@ private Project getValidProject(Long id) {
         return ResponseEntity.ok(sprints);
     }
 
+    @Override
+    public ResponseEntity<SprintSummaryDto> getSprintSummary(Long projectId, Long sprintId) {
+        Project project = getValidProject(projectId);
+        Sprint sprint = getValidSprint(sprintId);
+
+        if (!isUserRoleAdmin(projectId, userContext.getUserIdn()) || !isSprintInProject(sprint, project)) {
+            throw new ForbiddenAccessException();
+        }
+
+        List<AvailabilityDto> availabilities = projectService
+                .getAllUsersIdInProject(project)
+                .stream()
+                .map(id -> availabilityService.getBySprintIdAndUserId(sprintId, id).map(availabilityMapper::mapToApi)
+                        .orElseGet(
+                                () -> new AvailabilityDto().userId(id).timeAvailable(0L).timeRemaining(0L).notes("")))
+                .collect(Collectors.toList());
+
+        Long totalAvailableTime = availabilities
+                .stream()
+                .map(AvailabilityDto::getTimeAvailable)
+                .reduce(0L, Long::sum);
+
+        //TODO: Add MATH and ALGORITHMS to coefficient calculation
+        Double coefficient = 1D;
```

Comment:
mozna w sumie zrobic funkcje ktora ma rozsadna nazwe i zwraca na razie 1.
bedzie lepiej widac co jest do zrobienia, ale to taki szczegol przy okazji

Reply:
Zamykam, kod się zmienił. Co do coefficient to Julia zasugerowała, by implementacja algorytmu przeszła do następnego taska